torch/onnx/_internal/exporter.py

```diff
@@ -660,7 +663,48 @@ def __call__(self, *args: Any, options=None, **kwargs: Any) -> Any:
             for k, v in zip(ort_session.get_inputs(), onnx_input)
         }
 
-        return ort_session.run(None, onnxruntime_input)
+        onnxruntime_outputs = ort_session.run(None, onnxruntime_input)
+        # Update mutated buffers into the original PyTorch model
+        if isinstance(self._model_torch, torch.export.ExportedProgram):
+            self.update_mutated_buffers_onto_torch_model(
+                self._model_torch.module(), onnxruntime_outputs
+            )
+        return onnxruntime_outputs
+
+    def update_mutated_buffers_onto_torch_model(
+        self,
+        torch_model: torch.nn.Module,
+        onnxruntime_outputs: Sequence[
```

Comment:
```suggestion
        onnx_outputs: Sequence[
```
nit: doesn't have to be runtime specific?